api.js

```diff
@@ -27,8 +27,8 @@ const NAMES = require('./lib/metrics/names')
  * CONSTANTS
  *
  */
-const RUM_STUB =
-  "<script type='text/javascript' %s>window.NREUM||(NREUM={});" + 'NREUM.info = %s; %s</script>'
+const RUM_STUB = 'window.NREUM||(NREUM={}); NREUM.info = %s;'
```

Comment:
wow this was subtle but the reason the integration tests are failing.  There should not be a space between `;` and `NREUM.info`

Reply:
Nice catch, thanks a lot, I'll fix it :+1: